css counter descriptors: prefix and suffix (#29736)
mdn/content: files/en-us/web/css/@counter-style/suffix/index.md

{{CSSRef}}

> The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be appended at the end of the marker representation.

Reviewer: "appended" means "added to the end of", so this edit is not really needed. You could just follow the same style as you've used on the `prefix` page, and say "...that will be added to the end of..."

Author: I added it for non-native English speakers, but changing to added is even better!

> The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be appended at the end of the marker representation.

Author: ```suggestion
The **`suffix`** descriptor of the {{cssxref("@counter-style")}} rule specifies content that will be added to the end of the marker representation.
```

## Syntax

```css

> /* <symbol> values: string, image, or identifier  */

Reviewer: Same comment as for the `prefix` page. Would `<symbol> value` be better?

Author: dropped the "s"

> /* <symbol> values: string, image, or identifier  */

Author: ```suggestion
/* <symbol> value: string, image, or identifier  */
```
